### PR TITLE
Added New in Course-en.md

### PR DESCRIPTION
**Pull request by surya-mu**
## What does this PR do?
Adds a New course by Cisco Networking Academy on Javascript.

### Description
This course on Javascript is offered by Cisco Networking Academy for free and it allows the learner to algorithmically design, develop and write code on Javascript, and covering all the essentials and fundamentals for the same. 

### Why is this valuable (or not)?
This course helps is valuable because
* Understand a programmer’s work in the software development process and the role of fundamental development tools.
* Know how a program is interpreted and executed in an actual local and remote computer environment.

### How do we know it's really free?
This course is actually offered for free to all by Cisco, to help them learn and program concepts on Javascript and also to encourage students, accelerate careers, and also to kickstart your technology talents.

### For book lists, is it a book? For course lists, is it a course? etc.
This is a well structured Course offered by Cisco.

## Checklist:
- [☑️] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [☑️] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [☑️] Include author(s) and platform where appropriate.
- [☑️] Put lists in alphabetical order, correct spacing.
- [☑️] Add needed indications (PDF, access notes, under construction).
- [☑️] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -883,6 +883,7 @@
 * [JavaScript Coding Challenges](https://youtube.com/playlist?list=PLgBH1CvjOA63ROz8Wqd7RDD0qpvGXF8x5) - Florin Pop
 * [Javascript course](https://www.youtube.com/playlist?list=PLRAV69dS1uWSxUIk5o3vQY2-_VKsOpXLD) - Hitesh Choudhary
 * [Javascript Essentials](https://www.udemy.com/javascript-essentials/) - Lawrence Turton (Udemy)
+* [Javascript Essentials 1 JSE](https://www.netacad.com/courses/programming/javascript-essentials-1) - Cisco (Cisco Networking Academy)
```
Comment (eshellman) [inline courses/free-courses-en.md:886]:
```suggestion
* [Javascript Essentials 1 (JSE)](https://www.netacad.com/courses/programming/javascript-essentials-1) - (Cisco Networking Academy)
```

**Comment by surya-mu** [comment]
Made changes as mentioned, Please check thank you.

**Comment by eshellman** [comment]
you didn't like the parens around JSE?

**Comment by surya-mu** [comment]
Extremely Sorry, I didn't notice that change. I have added the parentheses now. Thanks for letting me know!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**